MM-30320 ALIAS DNS records

rachelbritton opened the pull request:
Added reference article and updated DNS Resource Records topic.

rachelbritton committed:
MM-30320 ALIAS DNS records

Added reference article and updated DNS Resource Records topic.

shersisson commented:
Looks good, maybe add a note that the full resource id is not displayed in the data column of an ALIAS record in the DNS records table in Blackstar, only the underlying record type and formatted resource id.

rachelbritton committed:
MM-30320. Added note and fixed link on DNS page

Added requested note about displayed resource ID and fixed formatting of link in user manual.

rachelbritton commented:
@shersisson Added a note as per your comment.

shersisson reviewed (approved):
looks good!

rachelbritton committed:
MM-30320. Fixed space after comma
